File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alpha_number = [i for i in range(97, 123) if chr(i) not in skip]
    answer = []
    for c in s:
        new_index = alpha_number.index(ord(c)) + index
        answer.append(chr(alpha_number[new_index % len(alpha_number)]))
    return "".join(answer)

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    let alpabet = [];
    for(let alpa = 97; alpa<= 122; alpa++) {
        var c = String.fromCharCode(alpa);
        if (!skip.includes(c)) { 
            alpabet.push(c);
        }
    }
    
    alpabet = alpabet.concat(alpabet).concat(alpabet);
    var answer = ""
    for(var c of [...s]) {
            answer += alpabet[alpabet.indexOf(c) + index];
    }
  
    return answer;
}